src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -119,9 +119,8 @@ the main body of the message as well as a quote.
 					<!-- Message Actions -->
 					<div
 						v-if="hasActions"
-						v-show="showActions"
 						class="message-body__main__right__actions"
-						:class="{ 'tall' : isTallEnough }">
+						:class="{ 'tall' : isTallEnough, 'action-show': showActions, 'action-hide': !showActions }">
```

Comment:
This doesn't solve the problem as the whole block is hidden if showActions is false

Reply:
The root cause of this bug is that block is display none, so the menu can't find exactly position of that block.
So I changed some logic here, when showActions false, this block will not show in UI, but not display none. ;)

Reply:
I can confirm that it works.
On that note, when the menu is open, it would be okay to not hide the action menu icon.

Reply:
I think that that's the way to go

Reply:
Even better, the actions emits an `update:open` event, so we could have

`<Actions  @update:open="handleActionMenuUpdate" />`

and then in the methods

```
handleActionMenuUpdate(velue) {
    this.isActionMenuOpen = value //This value being stored in the data
}
```

Reply:
Fixed

Reply:
You forgot to push?

Reply:
Seems like it